src/modules/taskClass.js

```diff
@@ -1,10 +1,10 @@
 const taskList = [];
 
 class Task {
-  constructor(description, completed = false, index = taskList.length) {
+  constructor(description, completed = false, id = taskList.length) {
     this.description = description;
     this.completed = completed;
-    this.index = index;
+    this.id = id;
```

Comment:
* [OPTIONAL]
* Sorry this wasn't pointed out by the previous reviewer 🥺 🙏  that is why i am making this OPTIONAL, but i strongly suggest you fix them as they are required  for this project 
* I think the property index be set to the value of the new array length  (i.e. if you're adding a 5th task to the list, the index of that task should equal 5) [Please refer to the project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=By%20default%20new%20tasks%20should%20have%20the%20property%20completed%20set%20to%20false%20and%20the%20property%20index%20set%20to%20the%20value%20of%20the%20new%20array%20length%20(i.e.%20if%20you%27re%20adding%20a%205th%20task%20to%20the%20list%2C%20the%20index%20of%20that%20task%20should%20equal%20to%205).) 
* HINT 💭 You can set your index by doing (index: x.length + 1 ) so that items created don't take the index `0`
* Don't forget to update the local storage for deleted items too (ie when item[ i ] is deleted item [ i  + 1] takes the index of item [ i ] ) because after making the change above the delete functionality would be affected  [check the project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Deleting%20a%20task%20should%20update%20all%20remaining%20items%27%20indexes%2C%20so%20they%20represent%20the%20current%20list%20order%20and%20are%20unique.)